src/spac/data_utils.py

```diff
@@ -507,7 +508,7 @@ def downsample_cells(input_data, annotations, n_samples=None, stratify=False,
         - stratify=False: Returns 'n_samples' for each unique value (or
           combination) of annotations.
         - stratify=True: Returns a total of 'n_samples' stratified by the
-          frequency of annotations.
+          frequency of the every label or combined label in the annotation(s).

```

Comment:
@fangliu117 , please remove "the" form "of the every label" and ass 's' in "combined labels"